.github/workflows/zfs-tests-sanity.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Indeed, explicit revision number gives more stability (Ubuntu 22.04 is coming soon). This is not strictly related to ASan/UBSan and since I'm touching the file, why not.

Reply:
The thought was to get problems as soon as we can in CI. But this is not the first time when somebody tries to make it explicit, so I won't be a stopper here.

Side note (sorry if I'm too pedantic) - it's easier to save context in PR/commit and review if changes are atomic and have one main theme. For example this change does look as a different topic for me.

Reply:
@gmelikov I'm lazy and I try to fix everything at once, it's good that there's my polar opposite in here :)
If the change is really bothersome, I'm ready to move it to another PR.

Reply:
I propose to leave it as `latest`, not all github actions are on latest so it would be better to have at least one job on it to get problems as fast as we can.